File: labs/J90_TemplateMethod/python/startingpoint/appropriation.py
from bill import Bill

class Appropriation(Bill):

    def execute_workflow(self):
        self._report("Proposal")
        self._report("Committee Debate")
        self._report("Committee Vote")
        self._report("Some other committee Debate")
        self._report("Some other committee Vote")
        self._report("Floor Debate")
        self._report("Floor Vote")
        self._report("Executive Signoff")

File: labs/J90_TemplateMethod/java/complete/app/src/main/java/TemplateMethodComplete/Appropriation.java
package TemplateMethodComplete;


public class Appropriation extends Bill {

  public Appropriation(String id) {
    super(id);
  }

  @Override
  public void postCommittee(){
    this.reportProgress("Some other committee debate.");
    this.reportProgress("Some other committee vote.");
  }
  
}
